int/config/check.go

```diff
@@ -0,0 +1,126 @@
+package config
+
+import (
+	"crypto/x509"
+	"fmt"
+	"path/filepath"
+
+	"github.com/massalabs/station/int/configuration"
+	"github.com/massalabs/station/pkg/certificate"
+	"github.com/massalabs/station/pkg/certificate/store"
+	"github.com/massalabs/station/pkg/logger"
+	"github.com/massalabs/station/pkg/nss"
+)
+
+// failureConsequence is the consequence of a failure to add the CA to NSS.
+const failureConsequence = "Station will only work using http, or you will have to add the CA to your browser manually."
+
+// Check performs a check on the configuration.
+func Check() error {
+	caRootPath, err := configuration.CAPath()
+	if err != nil {
+		return caNonBlockingError("failed to get CA path", err)
+	}
+
+	certPath := filepath.Join(caRootPath, configuration.CertificateAuthorityFileName)
+
+	err = checkCertificate(certPath)
+	if err != nil {
+		return nonBlockingError("Error while checking certificate", failureConsequence, err)
+	}
+
+	err = checkNSS(certPath)
+	if err != nil {
+		return nonBlockingError("Error while checking NSS", failureConsequence, err)
+	}
+
+	return nil
+}
+
+// nonBlockingError logs a non blocking error.
+func nonBlockingError(context string, consequence string, err error) error {
+	if err != nil {
+		logger.Warnf("%s: %s.", context, err)
+		logger.Warn(consequence)
+	}
+
+	return nil
+}
+
+// caNonBlockingError logs a non blocking error related to the CA loading failure.
```

Comment:
I would also add something about the `nil` always returned here since it uses `nonBlockingError` under the hood

Reply:
I think it's ok because thanks to the function name we understand that it will never block